example/src/app/a-bit-more-complex-component/child.component.less

```diff
@@ -0,0 +1,5 @@
+.kek { color: red }
+
+:host {
+  color: blue;
+}
```

Comment:
Is this file really required? Let's keep the test as simple as possible. Plus I think we do not have a preprocessor configured.

Reply:
I removed it. It was try to get nghost/ngcontent attributes in snapshots.